src/Microsoft.Sbom.Api/Output/Telemetry/IRecorder.cs

```diff
@@ -89,5 +89,10 @@ public interface IRecorder
     /// </summary>
     public Task FinalizeAndLogTelemetryAsync();
 
+    /// <summary>
+    /// Add an extra result in the form of a key-value pair to the telemetry.
+    /// </summary>
+    public void AddResult(string propertyName, string value);
```

Comment:
Since the interface is changing will this be a breaking change requiring a major version bump? If so this might be a good one to bring up at standup for awareness and planning.